packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -201,10 +203,16 @@ Please make sure you are running "shorebird init" from the root of your Flutter
       return ExitCode.software.code;
     }
 
-    _addShorebirdYamlToProject(appId, flavors: flavors);
+    _addShorebirdYamlToProject(
+      projectRoot: projectRoot,
+      appId: appId,
+      flavors: flavors,
+    );
 
     if (!shorebirdEnv.pubspecContainsShorebirdYaml) {
-      _addShorebirdYamlToPubspecAssets();
+      _addShorebirdYamlToPubspecAssets(
+        shorebirdEnv.getPubspecYamlFile(cwd: projectRoot),
```

Comment:
Makes me wonder if shorebirdEnv should have been created with a projectRoot (a later refactor).

Reply:
Yeah it probably can. I'll have a look in a subsequent PR 👍